pyproject.toml

```
@@ -14,17 +14,18 @@ classifiers = [
 ]
 description = "Ophyd devices and other utils that could be used across DLS beamlines"
 dependencies = [
-    "ophyd@git+https://github.com/DominicOram/ophyd@31a1762435bcb275d2555068233549885eab02e7", # Switch back to just "ophyd" once ophyd#1148, ophyd#1155, ophyd#1140 merged.
+    "ophyd-async",
+    "ophyd",
     "bluesky",
     "pyepics",
     "dataclasses-json",
     "pillow",
     "requests",
     "graypy",
     "pydantic<2.0",
-    "opencv-python-headless",                                                                  # For pin-tip detection.
-    "aioca",                                                                                   # Required for CA support with Ophyd V2.
-    "p4p",                                                                                     # Required for PVA support with Ophyd V2.
+    "opencv-python-headless", # For pin-tip detection.
+    "aioca",                  # Required for CA support with Ophyd V2.
+    "p4p",                    # Required for PVA support with Ophyd V2.
```

nit-pick: maybe we could replace the reference in these comments from "Ophyd V2" to "Ophyd async" now that the project has been renamed?

```suggestion
    "aioca",                  # Required for CA support with ophyd-async.
    "p4p",                    # Required for PVA support with ophyd-async.
```